wagtail/admin/templates/wagtailadmin/pages/confirm_move.html

```diff
@@ -6,11 +6,20 @@
     {% include "wagtailadmin/shared/header.html" with title=move_str subtitle=page_to_move.get_admin_display_title icon="doc-empty-inverse" %}
 
     <div class="nice-padding">
-        {% if page_to_move.is_leaf %}
-            <p>{% blocktrans trimmed with title=destination.get_admin_display_title %}Are you sure you want to move this page into '{{ title }}'?{% endblocktrans %}</p>
-        {% else %}
-            <p>{% blocktrans trimmed with title=destination.get_admin_display_title %}Are you sure you want to move this page and all of its children into '{{ title }}'?{% endblocktrans %}</p>
-        {% endif %}
+        <p>
+            {% if page_to_move.is_leaf %}
+                {% blocktrans trimmed with title=destination.get_admin_display_title %}Are you sure you want to move this page into '{{ title }}'?{% endblocktrans %}
+            {% else %}
+                {% blocktrans trimmed with title=destination.get_admin_display_title %}Are you sure you want to move this page and all of its children into '{{ title }}'?{% endblocktrans %}
+            {% endif %}
+            {% if translations_to_move_count %}
+                {% blocktrans trimmed count counter=translations_to_move_count %}
+                    This will also move {{ translations_to_move_count }} translation of this page and its child pages
```

Comment:
Don't we usually spell out "one" in these cases?

Reply:
I think we've used both. but updated in [21494a5](https://github.com/wagtail/wagtail/pull/7984/commits/21494a5a970719a528279a1b719f2c0a529e6ba3)